commands.md

```diff
@@ -12,82 +12,78 @@ Combine an array into a single string.
 
-A document can force parent groups to break by including `breakParent`
-(see below). A hard and literal line automatically include this so
-they always break parent groups. Breaks are propagated to all parent
-groups, so if a deeply nested expression has a hard break, everything
-with break. This only matters for "hard" breaks, i.e. newlines that
-are printed no matter what and can be statically analyzed.
+A document can force parent groups to break by including `breakParent` (see
+below). A hard and literal line automatically include this so they always break
+parent groups. Breaks are propagated to all parent groups, so if a deeply nested
+expression has a hard break, everything with break. This only matters for "hard"
+breaks, i.e. newlines that are printed no matter what and can be statically
+analyzed.
 
 For example, an array will try to fit on one line:
 
 ```js
-[1, "foo", { bar: 2 }]
+[1, "foo", { bar: 2 }];
 ```
 
-However, if any of the items inside the array have a hard break, the
-array will *always* break as well:
+However, if any of the items inside the array have a hard break, the array will
+_always_ break as well:
 
 ```js
 [
   1,
   function() {
-    return 2
+    return 2;
   },
   3
-]
+];
 ```
 
-Functions always break after the opening curly brace no matter what,
-so the array breaks as well for consistent formatting. See the
-implementation of `ArrayExpression` for an example.
+Functions always break after the opening curly brace no matter what, so the
+array breaks as well for consistent formatting. See the implementation of
+`ArrayExpression` for an example.
 
 ### conditionalGroup
 
-This should be used as **last resort** as it triggers an exponential complexity when nested. This will try to print the first argument, if it fit use it, otherwise go to the next one and so on.
+This should be used as **last resort** as it triggers an exponential complexity
+when nested. This will try to print the first argument, if it fit use it,
+otherwise go to the next one and so on.
 
 ```js
-conditionalGroup([a, b, c])
+conditionalGroup([a, b, c]);
 ```
 
 ### fill
 
-This is an alternative type of group which behave like text layout: it's going to add a break whenever the next element doesn't fit in the line anymore. The difference with a typical group is that it's not going to break all the separators, just the ones that are at the end of lines.
+This is an alternative type of group which behave like text layout: it's going
+to add a break whenever the next element doesn't fit in the line anymore. The
+difference with a typical group is that it's not going to break all the
+separators, just the ones that are at the end of lines.
 
 ```js
-fill(["I", line, "love", line, "prettier"])
+fill(["I", line, "love", line, "prettier"]);
 ```
 
-
 ### ifBreak
 
 Prints something if the current group breaks and something else if it doesn't.
 
 ```js
-ifBreak(";", " ")
+ifBreak(";", " ");
```

Comment:
Hmm, these are kind of expressions. Not statements, which is probably why there wasn't a semicolon here. Not the end of the world, I guess.